docs/Troubleshooting.md

```diff
@@ -36,6 +36,8 @@ You can configure the packager to use a port other than 8081 by using the `port`
 $ react-native start --port=8088
 ```
 
+If running on device from Xcode, you'll also need to update 8081 to your chosen port in `node_modules/react-native/React/React.xcodeproj/project.pbxproj`
```

Comment:
This is what is implied by the next line in the doc.

Can you instead add this note as a follow up to the next line in the doc?